core-services/prow/02_config/_config.yaml

```diff
@@ -508,6 +508,9 @@ tide:
     - release-4.5
     - release-4.6
     - openshift-4.1
+    - openshift-4.2
```

Comment:
@stevekuznetsov @petr-muller @droslean this will need your approval, ptal

also if you could check this config I have no idea why https://github.com/openshift/etcd/pull/25 currently reports `tide — In merge pool.` without any of the lgtm and approve tags being present. The excludeBranches here should apply on it, right?